.gitignore

```diff
@@ -63,9 +63,14 @@ node_modules/
 
 # react webapp gitignore
 webapp/node_modules
+webapp/dist
```

Comment:
You can add `dist/` to the top level. Never want to check in dist folders in an app.